Resources/Prototypes/Nyanotrasen/Entities/Clothing/Masks/masks.yml

```diff
@@ -28,3 +28,16 @@
     sprite: Nyanotrasen/Clothing/Mask/mask_oni_red.rsi
   - type: Clothing
     sprite: Nyanotrasen/Clothing/Mask/mask_oni_red.rsi
+
+- type: entity
+  parent: ClothingMaskPullableBase
+  id: felinidcone
+  name: Recovery cone
```

Comment:
casing for name would be "recovery cone". The game will capitalize it if it needs capitalized. Only capitalize proper nouns in the name field.